spec/support/document_type_field_examples.rb

```diff
@@ -0,0 +1,16 @@
+module DocumentTypeFieldExamples
```

Comment:
Can you help me understand the problem we're trying to solve here? It looks a bit like we're mandating which methods someone writes, but this should be self-evident by tests failing if such methods were missing. There does seem to be a duplication issue, so perhaps a little inheritance would clarify the interface and DRY things up. I don't think we should be adding meta-programming type tests, though.

Reply:
Fair 👍 I didn't think our tests were testing 'real world' usage of tags; I think I was misremembering what `topics { false }` was doing. So I anticipated us forgetting to implement, say, `pre_preview_issues` in one of our field models, and all of our tests continuing to pass.

I've just tried removing this meta-programming test and then commenting out `pre_preview_issues` for `TopicalEventsField`, and tests do indeed break, so it should be safe to remove this. I think the DRY-ing up I add later in the PR also helps with a lot of this.

Reply:
@ChrisBAshton I had the same thought about (lack of) coverage. I wrote another comment about how we could do this more explicitly (if we want to) with the approach we take to fix locale coverage.